hs_core/tasks.py

```diff
@@ -1,603 +1,601 @@
-    except SessionException:
-        # user may not have resources in user zone, so corresponding quota size AVU may not exist
-        # for this user
-        uqUserZoneSize = -1
-
-    if uqDataZoneSize < 0 and uqUserZoneSize < 0:
-        logger.error('no quota size AVU in data zone and user zone for the user ' + username)
-        return False
-    elif uqUserZoneSize < 0:
-        used_val = uqDataZoneSize
-    elif uqDataZoneSize < 0:
-        used_val = uqUserZoneSize
-    else:
-        used_val = uqDataZoneSize + uqUserZoneSize
-
-    uq.update_used_value(used_val)
-
-    return True
-
-
-@shared_task
-def update_web_services(services_url, api_token, timeout, publish_urls, res_id):
-    """Update web services hosted by GeoServer and HydroServer.
-
-    This function sends a resource id to the HydroShare web services manager
-    application, which will check the current status of the resource and register
-    or unregister services hosted by GeoServer and HydroServer.
-    The HydroShare web services manager will return a list of endpoint URLs
-    for both the resource and individual aggregations. If publish_urls is set to
-    True, these endpoints will be added to the extra metadata fields of the
-    resource and aggregations.
-    """
-    session = requests.Session()
-    session.headers.update(
-        {"Authorization": " ".join(("Token", str(api_token)))}
-    )
-
-    rest_url = str(services_url) + "/" + str(res_id) + "/"
-
-    try:
-        response = session.post(rest_url, timeout=timeout)
-
-        if publish_urls and response.status_code == status.HTTP_201_CREATED:
-            try:
-
-                resource = utils.get_resource_by_shortkey(res_id)
-                response_content = json.loads(response.content)
-
-                for key, value in response_content["resource"].iteritems():
-                    resource.extra_metadata[key] = value
-                    resource.save()
-
-                for url in response_content["content"]:
-                    logical_files = list(resource.logical_files)
-                    lf = logical_files[[i.aggregation_name for i in
-                                        logical_files].index(
-                                                    url["layer_name"].encode("utf-8")
-                                                )]
-                    lf.metadata.extra_metadata["Web Services URL"] = url["message"]
-                    lf.metadata.save()
-
-            except Exception as e:
-                logger.error(e)
-                return e
-
-        return response
-
-    except (requests.exceptions.RequestException, ValueError) as e:
-        logger.error(e)
-        return e
-
-
-@periodic_task(ignore_result=True, run_every=crontab(minute=00, hour=12))
-def daily_odm2_sync():
-    """
-    ODM2 variables are maintained on an external site this synchronizes data to HydroShare for local caching
-    """
-    ODM2Variable.sync()

```

Comment:
So. I'm open to input here.  the 2to3 tool writes the files as ASCII with CLRF endings.  After it does this, I rewrite all python files without CLRF endings.  There are, I believe 12 files that exist in the project that have CLRF endings already, this is one of them, so that's why it's showing up as a complete overwrite.  I could update it to still include CLRF endings, but I figured it makes sense to be consistent across the projects.  Since there are only 12, I found this a good time to make it consistent.

You can view the diff while ignore the whitespace change by passing w=1 as a query parameter to the diff